src/browser/window.ts

```diff
@@ -0,0 +1,15 @@
+import {ComputedStyle, Document, PrebootData, Window, Element} from '../common';
+
+export class WindowRef implements Window {
```

Comment:
Mock window here just used for testing purposes. It is the token for the window in the replayer which can be overriden with tests.